<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewRoundDialog</class>
 <widget class="QDialog" name="NewRoundDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonNormal">
       <property name="text">
        <string>Normal Round</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonHochzeit">
       <property name="text">
        <string>Hochzeit</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSolo">
       <property name="text">
        <string>Solo</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonTrumpfabgabe">
       <property name="text">
        <string>Trumpfabgabe</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::FieldsStayAtSizeHint</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Re:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="PlayersComboBox" name="comboBoxNormalRe1"/>
       </item>
       <item row="1" column="1">
        <widget class="PlayersComboBox" name="comboBoxNormalRe2"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Schweine:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="PlayersComboBox" name="comboBoxNormalSchweine"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Points:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinBoxNormalPoints">
         <property name="minimum">
          <number>-1000</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Comment:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QTextEdit" name="textEditNormalComment"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::FieldsStayAtSizeHint</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Hochzeit:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="PlayersComboBox" name="comboBoxHochzeitHochzeit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Fellow Player:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="PlayersComboBox" name="comboBoxHochzeitFellow"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Schweine:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="PlayersComboBox" name="comboBoxHochzeitSchweine"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Points:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinBoxHochzeitPoints">
         <property name="minimum">
          <number>-1000</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Comment:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QTextEdit" name="textEditHochzeitComment"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::FieldsStayAtSizeHint</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Solo:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="PlayersComboBox" name="comboBoxSoloPlayer"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Solo type:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBoxSoloType"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Schweine:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="PlayersComboBox" name="comboBoxSoloSchweine"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Points:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="spinBoxSoloPoints">
         <property name="minimum">
          <number>-1000</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Comment:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QTextEdit" name="textEditSoloComment"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Pflicht:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="checkBoxSoloPflicht">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Trumpfabgabe:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Accepted by:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="PlayersComboBox" name="comboBoxTrumpfabgabePlayer"/>
       </item>
       <item row="1" column="1">
        <widget class="PlayersComboBox" name="comboBoxTrumpfabgabeAccept"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Schweine:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="PlayersComboBox" name="comboBoxTrumpfabgabeSchweine"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Points:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Comment:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTextEdit" name="textEditTrumpfabgabeComment">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinBoxTrumpfabgabePoints">
         <property name="minimum">
          <number>-1000</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlayersComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">widgets/playerscombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewRoundDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewRoundDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
